fastly/origin_inspector_test.go

```diff
@@ -0,0 +1,34 @@
+package fastly
+
+import (
+	"testing"
+	"time"
+)
+
+func TestClient_GetOriginMetricsForService(t *testing.T) {
+	t.Parallel()
+
+	// NOTE: Update this to a recent time when regenerating the test fixtures, 
+	// otherwise the data may be outside of retention and an error will be
+	// returned.
```

Comment:
```suggestion
	// NOTE: Update this to a recent time when regenerating the test fixtures, 
	// otherwise the data may be outside of retention and an error will be
	// returned.
```